Dockerfile.preview

```diff
@@ -0,0 +1,16 @@
+# (c) Copyright 2019-2023 OLX
```

Comment:
What's the purpose of this file?

Reply:
The preview file does need to build the dali binary, as the test phase builds the dali binary. This Dockerfile is used to add the built dali binary to the alpine image, for the docker images that are created in the pull request branches